pydantic_extra_types/phone_numbers.py

```diff
@@ -107,9 +107,9 @@ class SomeModel(BaseModel):
             us_number: USNumberType
     """
 
-    default_region: Optional[str] = None
+    default_region: str | None = None
```

Comment:
I am not a maintainer but went into this PR from the referenced issue.

As far as I know `|` can be used as `Union` or `Optional` alternative only from python 3.10, see [Union Type](https://docs.python.org/3/library/stdtypes.html#union-type) and corresponding note at the end:

> Added in version 3.10.

Reply:
[PEP 563](https://peps.python.org/pep-0563/) is used here.

Reply:
Yeah, but `|` was proposed in [PEP 604](https://peps.python.org/pep-0604/), I don't see it in PEP 563

Reply:
The PEP 563 future statement stringifies type annotations by default. See the note in https://typing.readthedocs.io/en/latest/guides/modernizing.html#introduction explaining what it enables